src/components/AllArticles.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { fetchAllArticles } from "../api";
+import '../styles.css'
+
+const AllArticles = ({ user }) => {
+    const [list, setList] = useState([]);
+    const [isLoading, setLoading] = useState(true);  
+   
+  useEffect(() => {
+    fetchAllArticles().then((articles) => {
+      setList(articles);
+      setLoading(false);
+    });
+  }, [user]);
+
+    return (
+        <section id="articles">
+        <h2>All Articles</h2>
+        {isLoading ? (
+        <h3>Loading...</h3>
+      ) : (
+        <ul style={{paddingInline:0}}>
+          {list.map((article) => {
+                return (
+                    <li key={article.article_id} className="article">
+                        <img src={article.article_img_url} alt="article image" className="responsive-img"></img>
+                        <p><h4>Title: {article.title}</h4></p>
```

Comment:
there's a warning in the console about `<h4> cannot be a child of <p>` so just to keep our console free of warnings please change this to either p or h4 :)